src/patternfly/demos/Masthead/examples/Masthead.md

```diff
@@ -11,7 +11,7 @@ wrapperTag: div
 
 {{#*inline "masthead-template-content-toolbar-content"}}
   {{#> toolbar-content-section}}
-    {{#> toolbar-group toolbar-group--modifier="pf-m-align-right"}}
+    {{#> toolbar-group toolbar-group--modifier="pf-m-align-end"}}
```

Comment:
I'm not seeing these toolbars right aligned in the examples - the `.pf-m-align-end` class isn't defined - is this coming in the toolbar but isn't applied here yet?

Reply:
Yep, that's correct 
https://github.com/patternfly/patternfly/pull/6057/files#diff-0147a6969d342180a0d45586dcffd78147146a677926b982be70c9d6590bb5b7R31-R34